Makefile

```diff
@@ -71,7 +71,7 @@ test:
 
 proto:
 	protoc --gofast_out=plugins=grpc:. *.proto
-	protoc --gofast_out=plugins=grpc:. broker/*.proto
+	protoc --gogo_out=plugins=grpc:. broker/*.proto
```

Comment:
Just a reminder if it's necessary to update this in docs also:
https://github.com/mainflux/docs/blob/master/docs/dev-guide.md#protobuf

Reply:
@blokovi please send the PR

Reply:
Ok, after merging this PR

Reply:
right!

Reply:
@blokovi https://github.com/mainflux/docs/pull/25